app/models/move.rb

```diff
@@ -84,8 +84,9 @@ class Move < VersionedModel
   has_many :notifications, as: :topic, dependent: :destroy # NB: polymorphic association
   has_many :journeys, -> { default_order }, dependent: :restrict_with_exception, inverse_of: :move
   has_many :court_hearings, dependent: :restrict_with_exception
-  has_many :move_events, as: :eventable, dependent: :destroy # NB: polymorphic association
 
+  has_many :move_events, as: :eventable, dependent: :destroy # NB: polymorphic association
```

Comment:
what's the difference between `move_events` and `events` ? if they are the same thing (a v1 event), do we need both here?

Reply:
We need move_events because they're used in some outdated (to be removed in a separate PR) tests and in the diagnostics endpoint.